cli/src/pcluster/configure/easyconfig.py

```diff
@@ -367,16 +368,22 @@ def prompt_os(self):
 
     def prompt_instance_types(self):
         """Ask for master_instance_type and compute_instance_type (if necessary)."""
+        default_master_instance_type = self.cluster_section.get_param_value("master_instance_type")
+        if not default_master_instance_type:
+            default_master_instance_type = get_default_instance_type()
         self.master_instance_type = prompt(
             "Master instance type",
             lambda x: _is_instance_type_supported_for_head_node(x) and x in get_supported_instance_types(),
-            default_value=self.cluster_section.get_param_value("master_instance_type"),
+            default_value=default_master_instance_type,
```

Comment:
If pcluster_config is loaded with autorefresh we could directly put `self.cluster_section.get_param_value("master_instance_type")`

Reply:
related to this [thread](https://github.com/aws/aws-parallelcluster/pull/2254/files#r530417330)